src/main/java/com/speaker/service/validator/message/MessageDataValidatorImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.speaker.service.validator.message;
+
+import com.speaker.dto.MessageDTO;
+import com.speaker.dto.ValidatorError;
+import com.speaker.service.validator.type.ErrorType;
+import com.speaker.service.validator.type.FieldType;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.regex.Pattern;
+
+@Component
+@Primary
+@Qualifier("messageDataValidatorImpl")
+public class MessageDataValidatorImpl extends AbstractMessageCheckValidator {
+
+    @Override
+    public ValidatorError validate(String data) {
+        if (checkOrNull(data)) {
+            return ValidatorError.builder()
+                    .message(FieldType.DATE.getField() + ErrorType.EMPTY.getError())
+                    .field(FieldType.DATE.getField())
+                    .build();
+        }
+        if (splitByT(data).isEmpty()) {
```

Comment:
`splitByT` - bad name and unnecessary method, you can work with Date object for example